src/config/navigation/tabConfig.tsx

```diff
@@ -102,3 +106,31 @@ export const tabNavigatorConfig: TabNavigatorConfig = {
     aboutTabConfig
   ]
 };
+
+export const createDynamicTabConfig = (
+  accessibilityLabel: string,
+  iconName: string,
+  iconSize: number,
```

Comment:
`iconSize` should have a default value for cases, where nothing is configured like

```
{
  "label": "Custom",
  "screen": "VolunteerHome",
  "iconName": "Volunteer"
}
```

Reply:
yes you are right. I will give a value of 24 as standard. 👍